src/mutations/chatBot.ts

```diff
@@ -10,14 +10,15 @@ import { QueryClientConfig } from '../types';
 export default (queryConfig: QueryClientConfig) => {
   const { notifier } = queryConfig;
 
-  const usePostChatBot = () => {
+  // TODO: should use enum from the backend ?
```

Comment:
Yes I think it could be interesting to have it in SDK for example ? With that we have only one place to change if we decide to add a new version ?